app/components/navigation/navigation-item.tsx

```diff
@@ -12,9 +12,9 @@ export default function NavigationItem({ inputAttr, labelAttr, listAttr, childEl
         <label 
             htmlFor={inputAttr.id}
             id={labelAttr.id}
-            aria-controls={labelAttr.aria.controls}
+            aria-controls={labelAttr?.aria?.controls}
             data-testid={labelAttr.testId}
             className={labelAttr.classes }>{labelAttr.content}</label>
-        {hasSubpages && <ul className={listAttr.classes}  id={listAttr.id} data-testid={listAttr.testId} aria-hidden={isHidden} hidden={isHidden}>{childElements}</ul>}
+        {hasSubpages && <ul className={listAttr?.classes}  id={listAttr?.id} data-testid={listAttr?.testId} aria-hidden={isHidden} hidden={isHidden}>{childElements}</ul>}
```

Comment:
nit: is there a better way to format this line so we don't invoke a line wrap

Reply:
Yeah. I want to add new lines for props to our ESLint rules, btw